pkg/operator/controllers/routefix/routefix_controller.go

```diff
@@ -49,28 +48,33 @@ var (
 )
 
 // NewReconciler creates a new Reconciler
-func NewReconciler(log *logrus.Entry, arocli aroclient.Interface, configcli configclient.Interface, kubernetescli kubernetes.Interface, securitycli securityclient.Interface, restConfig *rest.Config) *Reconciler {
+func NewReconciler(log *logrus.Entry, arocli aroclient.Interface, configcli configclient.Interface, kubernetescli kubernetes.Interface, securitycli securityclient.Interface, dh dynamichelper.Interface) *Reconciler {
 	return &Reconciler{
 		log:           log,
 		arocli:        arocli,
 		configcli:     configcli,
 		kubernetescli: kubernetescli,
 		securitycli:   securitycli,
-		restConfig:    restConfig,
+		dh:            dh,
 		verFixed46:    verFixed46,
 		verFixed47:    verFixed47,
 	}
 }
 
 // Reconcile fixes the daemonset Routefix
 func (r *Reconciler) Reconcile(ctx context.Context, request ctrl.Request) (ctrl.Result, error) {
-	instance, err := r.arocli.AroV1alpha1().Clusters().Get(ctx, request.Name, metav1.GetOptions{})
+	instance, err := r.arocli.AroV1alpha1().Clusters().Get(ctx, arov1alpha1.SingletonClusterName, metav1.GetOptions{})
 	if err != nil {
-		r.log.Error(err)
```

Comment:
Reducing noise?

Reply:
yeah, there's not much point having it here if nowhere else